### ISSUE TITLE
ADMIN: Add Support for Recurring Events

### ISSUE THREAD

**Issue by meetulr** [PalisadoesFoundation/talawa-admin#1643]
### Is your feature request related to a problem? Please describe.

The CRUD functionality for recurring events was recently completed with this issue PalisadoesFoundation/talawa-api#1583 in the backend.
We need to integrate these features in the frontend too.

### Describe the solution you'd like

When creating recurring events, add an option to open a modal (like google calendar) for selecting a recurrence pattern.
When updating/deleting recurring events, add additional options, like choosing between updating/deleting `thisInstance` / `thisAndFollowingInstances` / `allInstances`.

### Additional context

> **For creation and management of recurring events, we're following [this](https://meetulr.github.io/Recurrence-Plan/) approach.**

**Comment by meetulr** [comment]
.

**Comment by meetulr** [comment]
.

**Comment by meetulr** [comment]
I apologize for the delay in addressing this issue.

After opening it, I moved on to complete other issues, and started researching and working on my proposals. That's why I haven't been able to advance my work on it this week.

I will start working on it and open the first PR next week.
Again, so sorry for the delay.

Thank you!

**Comment by palisadoes** [comment]
Can this be closed now?

**Comment by meetulr** [comment]
2 more PR's, i.e. for updating and deleting recurring events.😅

**Comment by palisadoes** [comment]
Should this be closed now?

**Comment by meetulr** [comment]
update part remaining, will finish it soon...

**Comment by meetulr** [comment]
Sorry for the delay here😅, this week has been unexpected, I wasn't able to work much😅.

I have been slowly working on it, and I've realized that the current implementation gets a little unintuitive and confusing, especially for updating recurring events. **I've got a nice fix for it.**

So, I would have to make backend changes too along with the frontend. I'll update the documentation afterwards.

I'll finish everything in 2 PRs, one in api and one in admin.
